### PR TITLE
[SPARK-10886] [Documentation] Random RDD creation example fix in MLlib statistics programming guide - mllib-statistics.md

### PR DESCRIPTION
**Pull request by jayantshekhar**
Fixed it to RandomRDDs.normalJavaRDD

### CODE REVIEW

File: docs/mllib-statistics.md [hunk 1]
```diff
@@ -521,7 +521,7 @@ val sc: SparkContext = ...
 
 // Generate a random double RDD that contains 1 million i.i.d. values drawn from the
 // standard normal distribution `N(0, 1)`, evenly distributed in 10 partitions.
-val u = normalRDD(sc, 1000000L, 10)
+val u = RandomRDDs.normalRDD(sc, 1000000L, 10)
```
Comment (srowen) [inline docs/mllib-statistics.md]: The symbol is already imported though; it works without the class name. The example should work.

File: docs/mllib-statistics.md [hunk 2]
```diff
@@ -542,7 +542,7 @@ JavaSparkContext jsc = ...
 
 // Generate a random double RDD that contains 1 million i.i.d. values drawn from the
 // standard normal distribution `N(0, 1)`, evenly distributed in 10 partitions.
-JavaDoubleRDD u = normalJavaRDD(jsc, 1000000L, 10);
+JavaDoubleRDD u = RandomRDDs.normalJavaRDD(jsc, 1000000L, 10);
```
Comment (srowen) [inline docs/mllib-statistics.md]: Same here. already static-imported.

**Comment by srowen** [comment]
Do you mind closing this PR?

**Comment by jayantshekhar** [comment]
Thanks Sean! Closing it.

**Pull request closed without merging**